### ISSUE TITLE
In Auto Forms mode, a fieldset's legend text is not announced at all if the fieldset appears as a descendant of an element with ARIA role="region"

### ISSUE THREAD

**Issue by jgauvreau** [FreedomScientific/VFO-standards-support#194]
### Summary
JAWS behavior for announcing interactive controls in a fieldset/legend is inconsistent. 
Case 1: Basic HTML fieldset
This is the most basic example, an HTML fieldset with a legend element as its descendant and then three explicitly labeled input elements. The legend element has simple text content of "Father's Name". The three pairs of label and input type="text" elements are explicitly associated using the for/id technique.

In IE 11 with JAWS 2019, the result is acceptable. When tabbing to each of the 3 form fields, JAWS announces the fieldset's legend text followed by the accessible name for the edit control. In contrast, test case 2 does NOT work as expected.

Case 2: ARIA region containing a HTML fieldset
This example expands upon Case 1 by adding a parent HTML div element with role="region". An aria-label attribute is defined on the div to provide an accessible name of "content panel" for the region. The region then contains an HTML fieldset with content similar to Case 1. (This scenario is to replicate the design pattern we've seen where a Dojo TitlePane contains a nested fieldset and JAWS fails to announce the fieldset's legend for any of the interactive control that are descendants of the fieldset.)

In IE 11 with JAWS 2019, the result is NOT Acceptable. When tabbing to each of the 3 form fields, JAWS fails to announce the fieldset's legend text for ANY of the edit controls; it only announces the accessible name for each edit.

Working Example:
1. Go to https://s.codepen.io/jlgauvreau/debug/89e54a09cc7bd8a6007765b3c377af4b
2. This test page contains four different code samples to compare what JAWS 2019 announces in Auto Forms Mode when tabbing to interactive controls (textboxes) that are descendants of a fieldset with legend. Each test case is described on the page along with my testing results using JAWS 2019 with IE 11 on a Windows 7 laptop.

### Expected result

Test Case 2: When in Forms mode, tabbing between interactive controls, I expected JAWS to announce the fieldset's legend text as a prefix to the accessible name of EACH interactive control in the fieldset (to mirror the experience from Test Case 1). 

### Actual result

It seems like JAWS 2019 uses different algorithms to announce the presence of a named fieldset group when using the virtual cursor versus Forms mode. The code samples in Case 1 and Case 3 produce an acceptable experience when testing with JAWS 2019 in IE 11. 

The best overall implementation seems to be the "hack"/workaround I implemented in Case 3 since in addition to yielding a concise listening experience with JAWS 2019 it also results in the most accurate UIA accessibility tree.

Test Case 1: Works as expected
Test Case 2: JAWS does not announce the fieldset's legend AT ALL. In IE 11 using JAWS 2019, when you tab and focus moves to an interactive control in the fieldset, JAWS does not announce the legend text at all. JAWS only announces the accessible name for each control followed by the control's role and then the cue "type in text".
Test Case 3: This implementation is a hack/workaround but arguably yields the best experience from a verbosity perspective. JAWS announces the fieldset's legend followed by the accessible name/role/value when focus moves to the FIRST interactive control in the fieldset group and then, for each subsequent control in the same fieldset, JAWS announces just the accessible name/role/value (without prefixing the fieldset's legend before the accessible name for the control).
Test Case 4: Similar to Case 3 but ultimately I eliminated this as a viable option since the UIA accessibility tree isn't as accurate as in Case 3.

**I welcome any and all questions or alternative suggestions to solve the problem I described for Test Case 2.**

### Additional Information
See additional test results documented in the CodePen at: https://s.codepen.io/jlgauvreau/debug/89e54a09cc7bd8a6007765b3c377af4b

#### JAWS version and build number
Latest version of JAWS (JAWS 2019.1904.60)
 
#### Operating System and version
Windows 7 Enterprise Service Pack 1

#### Browser and version:
Tested in multiple browser. Reported issue occurs in IE 11.0.9600.19326CO

**Comment by JAWS-test** [comment]
The problem occurs not only with role=region, but already, if an ancestor element has an explicit label (e.g. per aria-label). In the following example the div element: 

```
<!DOCTYPE html>
<html lang="en">
	<head>
		<meta charset="utf-8">
		<title>fieldset</title>
	</head>
<body>
	<div aria-label='DIV'>
		<fieldset>
			<legend>Mother</legend>
			<label>Name<input></label>
		</fieldset>
	</div>
</body>
</html>
```

**Comment by lamador** [comment]
The issue described in Test Case 2 was reproducible using the latest version of JAWS (2020.2004.66) and IE11 (11.836.18362.0) only when using the Tab key to navigate.  The "Mother's Name" group was identified when using the "Select a Form Field" window (Insert + F5) or using the "F" key to navigate to the form fields.

**Comment by stevefaulkner** [comment]
IE bug wontfix
